servers/rendering/renderer_rd/framebuffer_cache_rd.h

```diff
@@ -59,7 +59,7 @@ class FramebufferCacheRD : public Object {
 
 	static _FORCE_INLINE_ uint32_t _hash_pass(const RD::FramebufferPass &p, uint32_t h) {
 		h = hash_murmur3_one_32(p.depth_attachment, h);
-		h = hash_murmur3_one_32(p.vrs_attachment, h);
+		h = hash_murmur3_one_32(p.fragment_shading_rate_attachment, h);
```

Comment:
I think this should remain being named `vrs_attachment`, we're painting ourselves into a corner if Metal or DirectX or some other future extension also allows the density map to be specified per subpass, when we start naming variables in the `renderer_rd` based off of differences in Vulkan extensions.